io.catenax.manufacturing_capability/2.0.0/ManufacturingCapability.ttl

```diff
@@ -0,0 +1,176 @@
+
+:CapabilityEntity a samm:Entity ;
+   samm:extends :ElementAbstractEntity ;
+   samm:preferredName "Capability Entity"@en ;
+   samm:description "Element containing the bamm properties of a capability."@en ;
+   samm:properties ( :capabilityLabel ) .
+
+:CapabilityConstraintEntity a samm:Entity ;
+   samm:preferredName "Capability Constraint Entity"@en ;
+   samm:description "Element containing the bamm properties of a capability constraint."@en ;
+   samm:properties ( :references ) .
+
+:ElementAbstractEntity a samm:AbstractEntity ;
+   samm:preferredName "Element Abstract Entity"@en ;
+   samm:description "Abstract Entity containing the bamm properties every process, resource, product and capability entity shall contain."@en ;
+   samm:properties ( :propertySet ) .
+
+:hasInput a samm:Property ;
+   samm:preferredName "Has Input"@en ;
+   samm:description "Relation between a process and its input products."@en ;
+   samm:characteristic :ProductSetCharacteristic .
+
+:hasOutput a samm:Property ;
+   samm:preferredName "Has Output"@en ;
+   samm:description "Relation between a process and its output products."@en ;
+   samm:characteristic :ProductSetCharacteristic .
+
+:requires a samm:Property ;
+   samm:preferredName "Requires"@en ;
+   samm:description "Relation between a process and capabilities it requires."@en ;
+   samm:characteristic :CapabilitySetCharacteristic .
+
+:provides a samm:Property ;
+   samm:preferredName "Provides"@en ;
+   samm:description "Relation between a resource and capabilities it provides."@en ;
+   samm:characteristic :CapabilitySetCharacteristic .
+
+:productLabel a samm:Property ;
+   samm:preferredName "Product Label"@en ;
+   samm:description "Human readable label of a product, e.g. the name."@en ;
+   samm:characteristic samm-c:MultiLanguageText ;
+   samm:exampleValue "wheel suspension"@en .
+
+:capabilityLabel a samm:Property ;
+   samm:preferredName "Capability Label"@en ;
+   samm:description "Human readable label of a capability"@en ;
+   samm:characteristic samm-c:MultiLanguageText ;
+   samm:exampleValue "drilling"@en .
+
+:references a samm:Property ;
+   samm:preferredName "References"@en ;
+   samm:description "Relation between a capability constraint and its properties."@en ;
+   samm:characteristic :PropertySetCharacteristic .
+
+:propertySet a samm:Property ;
+   samm:preferredName "Property Set"@en ;
+   samm:description "Set of qualities or characteristics inherent in or ascribed to process, resource, product or capability entities."@en ;
+   samm:characteristic :PropertySetCharacteristic .
+
+:PropertySetCharacteristic a samm-c:Set ;
+   samm:preferredName "Property Set Characteristic"@en ;
+   samm:description "Characteristic for a set of property representations."@en ;
+   samm:dataType :PropertyEntity .
+
+:PropertyEntity a samm:Entity ;
+   samm:preferredName "Property Entity"@en ;
+   samm:description "Element containing the bamm properties of a property."@en ;
```

Comment:
do you really mean `bamm properties ` or `samm properties ` here?
If `samm`, then please change in other descriptions also. If not, just ignore it.

Reply:
Descriptions have been changed to `samm`